buildkite/test_plans.go

```diff
@@ -0,0 +1,48 @@
+package buildkite
+
+import "fmt"
+
+type TestPlansService struct {
+	client *Client
+}
+
+type Case struct {
+	Path              *string `json:"path,omitempty" yaml:"path,omitempty"`
+	EstimatedDuration *int `json:"estimated_duration,omitempty" yaml:"estimated_duration,omitempty"`
+}
+
+type Tests struct {
+	Format *string `json:"format,omitempty" yaml:"format,omitempty"`
+	Cases  []Case  `json:"cases,omitempty" yaml:"cases,omitempty"`
+}
```

Comment:
We've changed the schema. `Tests` is now just an array of `Case`.

Reply:
Ah yeah so we did, will need to update!